src/LabeledTextItem/LabeledTextItem.less

```diff
@@ -8,7 +8,7 @@
 
 	// Header
 	.moon-labeledtextitem-header {
-		display: inline-block;
+		display: block;
```

Comment:
Ah, didn't catch that either. We should override `display: inline-block` for `moon-expandable-list-item` then.